### ISSUE TITLE
Ctrl++ and Ctrl+- do not work on the Welcome page

### ISSUE THREAD

**Issue by aleksejrs** [CAFxX/ImageTweak#15]
ImageTweak
Firefox 7.0b2 or 6.0 on Debian GNU/Linux x86_64.

chrome://imagetweak/content/welcome.htm

Apparently that page is used as a demo of the extension, so Ctrl with "+" or "-" change the size of the icon in a frame. To change the Firefox’ zoom level, you have to use the View menu items.

Using the menu to change the zoom level is unusual and inconvenient for me, so I thought it was completely impossible to change the text size except by changing the font size preference. And changing the text size may be necessary (see my next report).

**Comment by CAFxX** [comment]
I am aware of this problem but I thought it wasn't a big deal (mostly because you can also use ctrl+mouse wheel to change the full page zoom). I could try to set by default a smaller zoom factor for the whole page if I detect that the screen is not big enough.

**Comment by aleksejrs** [comment]
I am using a graphics tablet with a wheel which is harder to rotate than it is to press a keyboard button.

I am not sure what you mean by "a smaller zoom factor" there, I guess you hadn’t read issue 16 before writing that.

**Comment by CAFxX** [comment]
Yeah, I thought that the problem was due to the screen being too small. I understand now that this isn't the case.
Unfortunately, as I wrote in the other issue, the current design of the page doesn't allow the text to scale (while preserving a proper layout).
